ui/decredmaterial/clickable_list.go

```diff
@@ -53,31 +66,41 @@ func (cl *ClickableList) handleClickables(count int) {
 
 func (cl *ClickableList) Layout(gtx layout.Context, count int, w layout.ListElement) layout.Dimensions {
 	cl.handleClickables(count)
-	return cl.List.Layout(gtx, count, func(gtx layout.Context, i int) layout.Dimensions {
-		if i == 0 { // first item
-			cl.clickables[i].Radius.TopLeft = cl.Radius.TopLeft
-			cl.clickables[i].Radius.TopRight = cl.Radius.TopRight
-		}
-		if i == count-1 { // last item
-			cl.clickables[i].Radius.BottomLeft = cl.Radius.BottomLeft
-			cl.clickables[i].Radius.BottomRight = cl.Radius.BottomRight
-		}
-		row := cl.clickables[i].Layout(gtx, func(gtx layout.Context) layout.Dimensions {
-			return w(gtx, i)
+	if cl.HideScroll {
+		return cl.List.Layout(gtx, count, func(gtx layout.Context, i int) layout.Dimensions {
+			return cl.row(gtx, count, i, w)
 		})
+	}
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return material.List(cl.theme.Base, cl.list).Layout(gtx, count, func(gtx C, i int) D {
```

Comment:
Why use `gio material` directly? Kindly add this as a file on the `decredmaterial` package. `list.go`.
See here for examples.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/checkbox.go.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/label.go.